src/Components/test/testassets/BasicTestApp/ServerReliability/ReliabilityComponent.razor

```diff
@@ -0,0 +1,35 @@
+@inject Microsoft.JSInterop.IJSRuntime JSRuntime
+@using System.Security.Cryptography;
```

Comment:
It would be good to remove this if we're not using it. We want it to be easy to audit use of crypto.

Reply:
Removed